docs/ccdproc/image_combination.rst

```diff
@@ -74,6 +74,19 @@ deviations below the median with
 
     A
 
+Extrema clipping
+++++++++++++++++
+
+For each pixel position in the input arrays, the algorithm will mask the
+highest `nhigh` and lowest `nlow` pixel values.  The resulting image will be a
```

Comment:
@joshwalawender -- the single backticks around nhigh, nlow, Nimages-nlow-nhigh, Nimages should be changed to double backticks. In Sphinx, singe backticks are references, double backticks are code font.

Looks like this was the only error in the travis build!